src/test/java/org/jabref/logic/shared/DBMSConnectionTest.java

```diff
@@ -2,30 +2,21 @@
 
 import java.sql.SQLException;
 
-import org.jabref.logic.shared.exception.InvalidDBMSConnectionPropertiesException;
 import org.jabref.model.database.shared.DBMSType;
 import org.jabref.testutils.category.DatabaseTest;
 
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 @DatabaseTest
```

Comment:
Does this really relies on a database being present? If not, then I would remove this annotation. If yes, then the paramaeteirzed test below probably needs to be converted to a static test using the enviroment variable.